src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categoryList: [],
+  status: 'Under Construction',
+};
+
+const categoriesSlice = createSlice(
+  {
+    name: 'category',
+    initialState,
+    reducers: {
+      checkStatus: (state) => state,
+    },
+  },
+);
```

Comment:
- If we check the project requirement from the screenshot below:

![image](https://github.com/Adams-Ghub/bookstore/assets/68030297/acea49db-d2e3-4a1c-bccc-fe6328ab0180)

- The second point conveys that there is a conditional check for the status, which means that if the status is "Under construction" then the reducer should return back "Under Construction", otherwise the reducer will return the categories array from the initial state👍🏼 

- [ ] With that being said please consider refactoring the categories slice reducer to include the previous logic 💪🏼